eras/dijkstra/impl/testlib/Test/Cardano/Ledger/Dijkstra/Imp/CertSpec.hs

```diff
@@ -0,0 +1,105 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE PatternSynonyms #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
+module Test.Cardano.Ledger.Dijkstra.Imp.CertSpec (spec) where
+
+import Cardano.Ledger.Conway.Governance (Voter (..))
+import Cardano.Ledger.Credential (Credential (..))
+import Cardano.Ledger.Dijkstra.Core (
+  EraTx (..),
+  EraTxBody (..),
+  emptyPParamsUpdate,
+  ppKeyDepositL,
+  ppuKeyDepositL,
+  pattern UnRegDepositTxCert,
+ )
+import Cardano.Ledger.Dijkstra.TxBody (DijkstraEraTxBody (..))
+import qualified Data.List.NonEmpty as NE
+import Data.Maybe.Strict (StrictMaybe (..))
+import qualified Data.OMap.Strict as OMap
+import qualified Data.Sequence.Strict as SSeq
+import qualified Data.Set as Set
+import Lens.Micro ((&), (.~))
+import Test.Cardano.Ledger.Dijkstra.ImpTest (
+  DijkstraEraImp,
+  ImpInit,
+  LedgerSpec,
+  expectStakeCredNotRegistered,
+  expectStakeCredRegistered,
+  freshKeyAddr,
+  freshKeyHash,
+  getsPParams,
+  impAnn,
+  passNEpochs,
+  registerInitialCommittee,
+  registerStakeCredential,
+  sendCoinTo,
+  setupSingleDRep,
+  submitFailingTx,
+  submitParameterChange,
+  submitTx_,
+  submitYesVoteCCs_,
+  submitYesVote_,
+ )
```

Comment:
This too. We don't use explicit imports for this module. If we start doing this, it will only result in tons of git conflicts.
```suggestion
import Test.Cardano.Ledger.Dijkstra.ImpTest
```